src/confluent_kafka/src/Admin.c

```diff
@@ -328,7 +328,94 @@ static int Admin_set_replica_assignment (const char *forApi, void *c_obj,
 
         return 1;
 }
+static int
+Admin_incremental_config_to_c(void *c_obj, PyObject *dict){
+        Py_ssize_t pos = 0;
+        PyObject *ko, *vo;
+        while(PyDict_Next(dict,&pos,&ko,&vo)){
+                PyObject *ks, *ks8;
+                const char *k;
+                const char *v = NULL;
+                int op = -1;
+                rd_kafka_error_t *error;
+                if (!(ks = cfl_PyObject_Unistr(ko))) {
+                        PyErr_Format(PyExc_ValueError,
+                                     "expected config name to be unicode "
+                                     "string");
+                        return 0;
+                }
+                k = cfl_PyUnistr_AsUTF8(ks, &ks8);
+
+                Py_ssize_t iter = 0;
+                PyObject *internal_ko,*internal_vo;
+                PyObject *internal_ks, *internal_ks8;
+                PyObject *internal_vs = NULL, *internal_vs8 = NULL;
```

Comment:
I think we can do better than internal in naming? Internal is confusing. Maybe `inner_dict` prefix would be more understandable?

I would say better overall naming.